### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.3`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/tymondesigns/jwt-auth/pull/2168/commits/c828e6579439c348ee8cdc969b8cbdb621eabac7?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/jwt-auth.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "tymon/jwt-auth": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4, 8.0]
-        laravel: [6.*, 7.*, 8.*]
+        php: [7.2, 7.3, 7.4, 8.0, 8.0]
```
Comment (mfn) [inline .github/workflows/phpunit.yml:17]: error, `8.0` is now twice

**Comment by mfn** [comment]
FTR, all these necessary changes are part of https://github.com/PHP-Open-Source-Saver/jwt-auth already

**Comment by jasonmccreary** [comment]
@mfn, yeah this came in a bit late as we realized some of the automation didn't work for packages where the repository was under a different name.

Feel free to close.

**Comment by mfn** [comment]
@jasonmccreary I've no stake / permissions here, can't close :-}

**Comment by mohamed-foly** [comment]
+1

**Comment by mohamed-foly** [comment]
I forked and applied the required updates 
`composer require mohamed-foly/jwt-auth:dev-laravel-9`
https://packagist.org/packages/mohamed-foly/jwt-auth#dev-laravel-9

**Comment by mfn** [comment]
FTR, as mentioned in https://github.com/tymondesigns/jwt-auth/pull/2168#issuecomment-1037040160 , this is already part of a community fork ❗

**Pull request closed without merging**